flow-bridge/src/main/java/org/reactivestreams/ReactiveStreamsFlowBridge.java

```diff
@@ -188,15 +202,15 @@ public void cancel() {
      */
     static final class FlowToReactiveSubscriber<T>
             implements Flow.Subscriber<T> {
-        private final org.reactivestreams.Subscriber<? super T> reactiveStreams;
+        public final org.reactivestreams.Subscriber<? super T> reactiveStreams;
```

Comment:
yeah, saw the need for exposing it, but would rather go "java style" since we're java project, so getter.
I'd call it `underlying()` or `delegate()` i guess?

Reply:
I can make it package-protected instead?